assets/components/contributionSelection/helpers.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { config as contributionConfig } from 'helpers/contributions';
+
+import type { Contrib as ContributionType } from 'helpers/contributions';
+import type { Radio } from 'components/radioToggle/radioToggle';
+import type { Currency } from 'helpers/internationalisation/currency';
+import type { IsoCountry } from 'helpers/internationalisation/country';
+
+
+// ----- Setup ----- //
+
+const spokenCurrencies = {
+  GBP: {
+    singular: 'pound',
+    plural: 'pounds',
+  },
+  USD: {
+    singular: 'dollar',
+    plural: 'dollars',
+  },
+};
+
+const amounts = {
+  ONE_OFF: {
+    GBP: [
+      { value: '25', spoken: 'twenty five' },
+      { value: '50', spoken: 'fifty' },
+      { value: '100', spoken: 'one hundred' },
+      { value: '250', spoken: 'two hundred and fifty' },
+    ],
+    USD: [
+      { value: '25', spoken: 'twenty five' },
+      { value: '50', spoken: 'fifty' },
+      { value: '100', spoken: 'one hundred' },
+      { value: '250', spoken: 'two hundred and fifty' },
+    ],
```

Comment:
Since the data types changed in master (`AUD` was added) I would merge/rebase master to this branch before merging. If not, I believe the checks in master will fail after merging.

Reply:
Done, and updated the other default amounts to match results of AB tests (e.g. 2, 5, 10 for monthly).